frontend/runtime_configs/rdev.env

```diff
@@ -1,4 +1,4 @@
 RUNTIME_VAR_API_URL=$API_URL
-RUNTIME_VAR_CELLGUIDE_DATA_URL=https://cellguide.cellxgene.staging.single-cell.czi.technology
+RUNTIME_VAR_CELLGUIDE_DATA_URL=https://cellguide.cellxgene.dev.single-cell.czi.technology
```

Comment:
This needs to be `dev` as `env-rdev-cellguide/` is in s3://cellguide-data-public-dev.